src/main/java/bitflow4j/steps/normalization/AbstractOnlineScaler.java

```diff
@@ -25,6 +25,22 @@
         List<Boolean> conceptChanged(AbstractOnlineScaler scaler, List<String> changedFeatures);
     }
 
+    public interface ConceptChangeDetector {
+        boolean isConceptChanged(double value);
+
+        static ConceptChangeDetector getInstance(String type, double threshold) {
```

Comment:
Where is this method used? (sorry, did not check in IDE).

Reply:
OnlineAutoMinMaxScaler and ScaleMinMaxWithModelRepository